docs/styles.md

```diff
@@ -1267,6 +1269,50 @@ Any element that contains text.
 
 ---
 
+## Black Text
+
+This class should be used to apply the color black to text.
+
+**Example**
+
+<div style="text-align: center;">
+    <img src="./static/blacktext.png" width="500">
+</div>
+
+**Availability**
+
+Any element that contains text.
+
+**Usage**
+
+```html
+<p class="os-raise-black-text">This color of this text is black.</p>
+```
+
+---
+
+## Green Bold Text
+
+This class should be used to apply the following to text: green, bold, and 24px.
+
+**Example**
+
+<div style="text-align: center;">
+    <img src="./static/ctatextstyle.png" width="500">
+</div>
+
+**Availability**
+
+Any element that contains text and is part of CTA block content.
+
+**Usage**
+
+```html
+<p class="os-raise-cta-title-style-theme">This text is green, bold, and 24px.</p>
```

Comment:
I think you may have missed updating the HTML here with the name change.

Reply:
I did - must have misspelled when I did my search across the files.